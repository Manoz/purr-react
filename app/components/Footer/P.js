import styled from 'styled-components';

const P = styled.p`
font-size: 15px;
  letter-spacing: 0;
`;

export default P;
