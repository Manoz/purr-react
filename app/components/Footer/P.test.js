import React from 'react';
import { shallow } from 'enzyme';

import P from './P';

describe('<P />', () => {
  it('should render a <p> tag', () => {
    const wrapper = shallow(<P>test</P>);
    expect(wrapper).toMatchSnapshot();
  });

  it('should have children', () => {
    const wrapper = shallow(<P>test</P>);
    expect(wrapper.contains('test')).toEqual(true);
  });
});
