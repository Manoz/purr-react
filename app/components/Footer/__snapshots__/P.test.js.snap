// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<P /> should render a <p> tag 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "P-sc-1wt57lm-0",
        "isStatic": true,
        "rules": Array [
          "font-size:15px;letter-spacing:0;",
        ],
      },
      "displayName": "P",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "P-sc-1wt57lm-0",
      "target": "p",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  test
</StyledComponent>
`;
